### ISSUE TITLE
What to modify for multi-label classification in Colab Notebook?

### ISSUE THREAD

**Issue by rohanali1798** [ArmDeveloperEcosystem/ml-audio-classifier-example-for-pico#4]
Hi can you please guide us on how to modify the colab notebook to make it work for multi-classification please? 
TIA.

**Comment by sandeepmistry** [comment]
Hi @rohanali1798,

You would have to change the classification head from:
```python
classifier_head = tf.keras.layers.Dense(1, activation="sigmoid")(model_body.output)
```
to
```python
# for 3 classes
classifier_head = tf.keras.layers.Dense(3, activation='softmax')(model_body.output)
```

in the transfer learning section, then ensure the training data has the appropriate labels for the multi-classification labels, and train with a `SparseCategoricalCrossentropy` loss function (instead of `BinaryCrossentropy`)

```
fine_tune_model.compile(
    optimizer=tf.keras.optimizers.Adam(learning_rate=0.01),
    loss=tf.keras.losses.BinaryCrossentropy(from_logits=False),
    metrics=METRICS,
)
```

For the inference side, https://github.com/ArmDeveloperEcosystem/ml-audio-classifier-example-for-pico/issues/1#issuecomment-980029385 has some tips.

**Comment by rohanali1798** [comment]
Great. Thanks! I will try this and let you know.
